stock-tracker/src/features/renderWithRedux.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { createStore, Store } from "redux";
+import { rootReducer, AppState } from "../store/rootReducer";
+import { render } from "@testing-library/react";
+import { Provider } from "react-redux";
+
+const initialState = {};
+
+interface Options {
+  state?: AppState;
+  store?: Store;
+}
+
+export function renderWithRedux(
+  component: React.ReactElement,
+  options: Options = {}
+) {
+  const {
+    state = initialState,
```

Comment:
Shouldn't the default `state` value be `undefined`? :thinking: 

cc @voliva

Reply:
Yep, good catch - So you don't really need to set a default value for it (as by default it's already undefined).

createStore calls the rootReducer to build the "initialState".